internal/webpack/configFactory.js

```diff
@@ -212,6 +219,11 @@ export default function webpackConfigFactory(buildOptions) {
           }),
       ),
 
+      // Implement webpack 3 scope hoisting that will remove function wrappers
+      // around your modules you may see some small size improvements. However,
+      // the significant improvement will be how fast the JavaScript loads in the browser.
+      ifClient(new webpack.optimize.ModuleConcatenationPlugin()),
```

Comment:
I'd argue this should be used only in production builds since it impacts build speed. Maybe I'm wrong?